Convergence plot test fails when run locally in VSCode

**Describe the bug**
<!-- A clear and concise description of what the bug is -->
`test_convergence_plot_command_line` fails because the RuntimeError does not raise.

```
    def test_convergence_plot_command_line(
        config_verysimple, atomic_dataset, monkeysession
    ):
        monkeysession.setattr(
            "tardis.util.environment.Environment.is_notebook",
            lambda: False,
        )
        atomic_data = deepcopy(atomic_dataset)
>       with pytest.raises(RuntimeError):
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
E       Failed: DID NOT RAISE <class 'RuntimeError'>

tardis/visualization/tools/tests/test_convergence_plot.py:230: Failed
```

**To Reproduce**
<!-- Steps to reproduce the behavior, link to a notebook or a copy-pastable example -->
Run the test in VSCode using its test interface.

**Screenshots**
<!-- If applicable, add screenshots to help explain your problem -->

**System**
 - OS:
    - [x] GNU/Linux
    - [ ] macOS

 - Environment (`conda list`):

**Additional context**
<!-- Add any other context about the problem here -->

--- a/tardis/visualization/tools/tests/test_convergence_plot.py
+++ b/tardis/visualization/tools/tests/test_convergence_plot.py
@@ -223,7 +223,7 @@ def test_convergence_plot_command_line(
     config_verysimple, atomic_dataset, monkeysession
 ):
     monkeysession.setattr(
-        "tardis.util.environment.Environment.is_notebook",
+        "tardis.util.environment.Environment.allows_widget_display",
         lambda: False,
     )
     atomic_data = deepcopy(atomic_dataset)

--- a/tardis/visualization/tools/tests/test_convergence_plot.py
+++ b/tardis/visualization/tools/tests/test_convergence_plot.py
@@ -220,9 +220,9 @@ def test_override_plot_parameters(convergence_plots):
 
 
 def test_convergence_plot_command_line(
-    config_verysimple, atomic_dataset, monkeysession
+    config_verysimple, atomic_dataset, monkeypatch
 ):
-    monkeysession.setattr(
+    monkeypatch.setattr(
         "tardis.util.environment.Environment.allows_widget_display",
         lambda: False,
     )

